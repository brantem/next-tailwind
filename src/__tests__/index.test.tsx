import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

import Home from 'pages';

test('Home', () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
  expect(screen.queryByText('Hello')).toBeInTheDocument();
});
