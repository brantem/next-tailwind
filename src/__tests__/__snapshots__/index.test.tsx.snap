// Vitest Snapshot v1

exports[`Home 1`] = `
<div>
  <h1>
    Hello
  </h1>
</div>
`;
